samples/bench-actor/README.md

```diff
@@ -0,0 +1,21 @@
+# New Actor Template
```

Comment:
We should probably update this README to at least just have a different heading, and then a small description about this being a benchmark actor

Reply:
Sigh. I had this readme updated locally and just hadn't pushed it.